### PR TITLE
AP_Mount: Alexmos: increase precision on parsed angle

### PR DESCRIPTION
**Pull request by MattKear**
Noticed this issue when helping someone out.  We are using integer math when calculating the angle in degrees from the provided value.  This minor rework helps to maintaining a little more accuracy so that we don't round to integer degrees.

I have also done a little tidying by moving the defines to the .cpp.

### CODE REVIEW

File: libraries/AP_Mount/AP_Mount_Alexmos.cpp [hunk 1]
```diff
@@ -3,6 +3,57 @@
...
+#define AP_MOUNT_ALEXMOS_MODE_NO_CONTROL 0
+#define AP_MOUNT_ALEXMOS_MODE_SPEED 1
+#define AP_MOUNT_ALEXMOS_MODE_ANGLE 2
+#define AP_MOUNT_ALEXMOS_MODE_SPEED_ANGLE 3
+#define AP_MOUNT_ALEXMOS_MODE_RC 4
+
+#define AP_MOUNT_ALEXMOS_SPEED 30 // deg/s
+
+#define INT14_DEGREES = (360.0f / 0x3FFF) // 1 full rotation in degrees over 14 bit range
+#define VALUE_TO_DEGREE(d) (float(d)*INT14_DEGREES)
```
Comment (MattKear) [inline libraries/AP_Mount/AP_Mount_Alexmos.cpp:53]: This is the key bit that has changed to improve the angle precision

**Review by rmackay9 — APPROVED** [review]

**Comment by rmackay9** [comment]
This looks OK to me and it has been tested of course?

**Comment by MattKear** [comment]
A client is testing this for me.  I will report back once the testing has shown the improvement works.

**Comment by IamPete1** [comment]
Before we get binned value, although not exactly integers:

image

After we get much smoother:
image

**Pull request merged**